storybookjs/storybook
# Docs: Fix spelling in ArgTypes doc block image

Closes #
N/A

## What I did

Adjusts the spelling of the word "principal" (previously "principle") for grammatical correctness. This image appears on the ArgTypes doc page: https://storybook.js.org/docs/api/arg-types

This image matches the original's color profile, alpha channel setting, and dimensions exactly. It also shaves off a couple of kilobytes from the file size.

## Checklist for Contributors

### Testing

#### The changes in this PR are covered in the following automated tests:

N/A

#### The changes in this PR are covered in the following automated tests:

#### Manual testing

Use GitHub's onion skin or swipe features to see that no visual differences beyond the spelling correction exist between images.

### Documentation

N/A

      [MIGRATION.MD](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md)

## Checklist for Maintainers

   <details>
     <summary>Available labels</summary>

     - `bug`: Internal changes that fixes incorrect behavior.
     - `maintenance`: User-facing maintenance tasks.
     - `dependencies`: Upgrading (sometimes downgrading) dependencies.
     - `build`: Internal-facing build tooling & test updates. Will not show up in release changelog.
     - `cleanup`: Minor cleanup style change. Will not show up in release changelog.
     - `documentation`: Documentation **only** changes. Will not show up in release changelog.
     - `feature request`: Introducing a new feature.
     - `BREAKING CHANGE`: Changes that break compatibility in some way with current major version.
     - `other`: Changes that don't fit in the above categories.
   
   </details>
